src/lib.nix

```diff
@@ -200,8 +200,29 @@
       (allOutputs: output: lib.recursiveUpdate allOutputs output)
       {}
       flakifiedOutputsList;
+
+    errorFlakeOutputs = {
+      apps =
+        l.mapAttrs
+        (system: pkgs: {
+          detect-projects =
+            dream2nixFor.${system}.flakeApps.detect-projects;
+
+          error = throw (missingProjectsError source);
+        })
+        allPkgs;
+    };
+
+    finalOutputs = let
+      givenProjects = args.projects or {};
+    in
+      if
+        (givenProjects == {} && autoProjects == false)
+        || (l.isPath givenProjects && ! l.pathExists givenProjects)
+      then errorFlakeOutputs
```

Comment:
I think we should replace the `error` app with `builtins.warn (missingProjectsError source) errorFlakeOutputs` here. It would be more prevalent since it would be showed with every command.